manager/dispatcher/assignments.go

```diff
@@ -45,39 +34,83 @@ func newAssignmentSet(log *logrus.Entry, dp *drivers.DriverProvider) *assignment
 	}
 }
 
+func assignSecret(a *assignmentSet, readTx store.ReadTx, mapKey typeAndID) {
+	a.tasksUsingDependency[mapKey] = make(map[string]struct{})
+	a.log.WithField("resource.type", "secret")
```

Comment:
`WithField` returns a logger with that field added to it - think of it as functional programming. You need to use the return value later on in the function, otherwise the call does nothing.